### PR TITLE
Issue 52 - fix CI Travis unit tests

### PR DESCRIPTION
**Pull request by PopGoesTheWza**
cf. #52

### CODE REVIEW

**Comment by PopGoesTheWza** [comment]
CI Travis unit tests now pass fine. The linting is still failing but that issue is covered with PR #51

File: packages/it/index.js [hunk 1]
```diff
@@ -35,24 +35,24 @@ function standardiseDescriptor(descr) {
   var deepConf = descr.deepConf;
 
   var p = isObject(props) || isObject(properties) ?
-    assign({}, props, properties) : undefined;
+    assign(assign({}, props), properties) : {};
```
Comment (koresar) [inline packages/it/index.js]: I remembered now. This (and similar) change is against the specification. I have to reject this PR unfortunately. The root cause is different.

File: packages/core/merge.js [hunk 2]
```diff
@@ -27,7 +27,7 @@ function mergeOne(dst, src) {
     // is this a regular property?
     if (desc.hasOwnProperty('value')) { // eslint-disable-line
       // Do not merge properties with the 'undefined' value.
-      if (desc.value === undefined) return;
```
Comment (koresar) [inline packages/core/merge.js]: Basically, hat this line is saying:

> This code
> ```js
> merge(anything, undefined)
> ```
> must do thing.

I believe the right fix would be to replace `return` with `continue`. Although, I'm not sure. I don't have time to check that. Sorry.

Comment (koresar) [reply packages/core/merge.js]: Looking at this line:

https://github.com/stampit-org/stampit/blob/5b4cbbeb351e40edf3bd9c5e13d3c2d45c97581a/src/stampit.js#L99

I believe `continue` is the right way.

File: packages/it/index.js (hunk 1 above)
Comment (PopGoesTheWza) [reply packages/it/index.js]: Just to clarify, the above line attempt to addreeses two issues:

1. the `assign()` function currently accepts two parameters, thus (unless `assign()` is to be refactored) in order to process three or more parameters, calls to `assign()` need to be nest by pair of parameters.
1. the last parameters to be `undefined` or `{}` for which I should probably investigate more.

Do you agree about the first point and reject the second?

Comment (PopGoesTheWza) [reply packages/it/index.js]: @koresar I believe I have worked out the correct fix now.

```
  var sdp = isObject(deepStatics) ? merge({}, deepStatics) : undefined;
  sdp = isObject(staticDeepProperties) ? merge(sdp || {}, staticDeepProperties) : sdp;
```

Currently `merge()` throws when the first argument is undefined, hence the `sdp || {}`

Also wondering if `merge({}, deepStatics)` couldn't be simplified with `Object.assign(deepStatics)`

**Pull request closed without merging**